crates/bevy_render/src/render_resource/pipeline_cache.rs

```diff
@@ -865,6 +917,27 @@ impl PipelineCache {
     }
 }
 
+fn create_pipeline_task(
+    task: impl Future<Output = Result<Pipeline, PipelineCacheError>> + Send + 'static,
+) -> CachedPipelineState {
+    #[cfg(all(
+        not(target_arch = "wasm32"),
+        not(target_os = "macos"),
```

Comment:
Is there any documentation on why this isn't available on macOS - https://docs.rs/wgpu/latest/wgpu/struct.Device.html#method.create_compute_pipeline doesn't document why this would be avoided

(I'm looking at this for an external project, not Bevy, but I'm trying to understand prior art)